cgmes/cgmes-model/src/main/java/com/powsybl/cgmes/model/triplestore/CgmesModelTripleStore.java

```diff
@@ -688,6 +703,8 @@ private String injectParams(String queryText, String... params) {
     private final int cimVersion;
     private final TripleStore tripleStore;
     private final QueryCatalog queryCatalog;
+    private boolean nodeBreaker;
+    private boolean nodeBreakerComputed = false;
```

Comment:
Rather than adding an intermediary boolean, you can use `Boolean` and compute it when it is null

Reply:
Done